bundles/action/org.openhab.action.slack/README.md

```diff
@@ -0,0 +1,49 @@
+# Slack Action
+The Slack action sends messages to a Slack ([https://slack.com/]) channel or Slack user from a rule or script.
```

Comment:
Since this page will eventually be shown on docs.openhab.org, it has to conform to the markdown rendered used there as well as here on GitHub.  So every heading has to have a blank line above and below, and every code block surrounded by \`\`\` has to have a blank line before the beginning and a blank line after the end.